nautobot/virtualization/tests/test_api.py

```diff
@@ -326,3 +327,54 @@ def setUpTestData(cls):
                 "untagged_vlan": vlans[2].pk,
             },
         ]
+
+
+class VMInterfaceTestVersion13(VMInterfaceTestVersion12):
+    api_version = "1.3"
```

Comment:
```suggestion
class VMInterfaceTestVersion14(VMInterfaceTestVersion12):
    api_version = "1.4"
```

This will begin to be available in 1.4. More places need to be updated